meilisearch-release-changelog/v1.0.0.md

```diff
@@ -0,0 +1,121 @@
+
+We are proud to announce we are releasing the first stable release of Meilisearch! 🎉 This new release, contrary to v0.30.0, focus on stabilization instead of bringing a lot of new features. We took the opportunity to make the final touch regarding some breaking changes in the CLI but also in our error handler.
+Since Meilisearch is now stable, we will not introduce any new breaking changes in the future versions of Meilisearch until releasing v2.0.0. Here you have more information about what we consider [as breaking for Meilisearch](https://github.com/meilisearch/engine-team/blob/main/resources/versioning-policy.md).
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+🧰 Most integrations (SDKs, clients, libraries) will be compatible with this version within four hours of the release. Sometimes this can take up to 48 hours, depending on the issues we encounter during the release.
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+Here is the exhaustive list of integrations  **not immediately compatible** with v1.0.0:
+- TBD
+
+<!-- The following line should NOT be put in the official release changelogs -->
+📖 Meilisearch's official documentation will be ready for the official v1.0.0 release. Meanwhile, work-in-progress documentation is available [here]().
+
+# New features and improvements 🔥
+
+# Charabia
+
+TBD
+
+# Generate master key option
+
+You can now activate the `--generate-master-key` CLI option when launching Meilisearch.
+
+This option is particular since it will not launch Meilisearhc but will generate a safe-to-use master key for Meilisearch and will exit without any error.
+
+Done in #3274 by @dureuill
+
+## Other improvements
+
+* Re-enable importing from dumps v1: you can now migrate from Meilisearch v0.20.0 and older to the latest version using a dump (#3170) @dureuill
+* Improve [`download-latest.sh`](https://github.com/meilisearch/meilisearch/blob/main/download-latest.sh) script: integrate apple-silicon binary (#3169) @curquiza
+* Clarify error message when the DB and engine versions are incompatible (#3279) @dureuill
+* Search speed improvements:
+  * synonyms (TBD)
+  * ranking rules (TDB)
+  * ... (TBD)
+
+# Breaking changes ⚠️
+
+Since we are releasing v1.0.0, this is the last time we are introducing breaking changes to this release before v2.0.0.
+
+## New CLI behavior
+
+* For security reasons, Meilisearch will now reject master keys that are less than 16 bytes (#3274) @dureuill
+* Remove `--max-index-size` and `--max-task-db` configuration options (#3278) @dureuill
+Indeed, these options did not include the real space Meilisearch takes on the disk and were not adapted to the user's needs.
```

Comment:
Docs team will help us

Reply:
I wonder if we should maybe link to the [tradeoff discussion](https://github.com/meilisearch/meilisearch/discussions/3280).

Maybe...

```suggestion
These options were not serving their intended purpose of limiting the disk space taken by Meilisearch, for the reasons laid out [here](https://github.com/meilisearch/product/discussions/567#discussioncomment-4322126), but actually articulate a [tradeoff](https://github.com/meilisearch/meilisearch/discussions/3280) between the number of indexes and their maximum size that we'd like to express differently in the future.

In v1, the number of indexes is limited to around 200 under Linux/macOS and around 20 under Windows, with a maximum size of 500GiB per index. We plan to lift these limitations in future versions, please [reach out](https://github.com/meilisearch/meilisearch/discussions/3280) if the new limits impede your usage.
```

Some of this might be out of place in the changelog 🤔... in any case, I'd like this communicated to the users.

Reply:
Hum really complex to me, and not really interesting for the users reading the changelog. The docs are more appropriate I think. Maybe a link to the docs could help?

However, maybe we could have some words if some people were using it and what they could do instead.

Reply:
> However, maybe we could have some words if some people were using it and what they could do instead.

Not much. Reach out to us or limit disk space from their infrastructure.

Reply:
I think we can go with @dureuill's suggestion, but maybe shorten it to: 
```suggestion
These options were not serving their intended purpose of limiting the disk space taken by Meilisearch for the reasons laid out [here](https://github.com/meilisearch/product/discussions/567#discussioncomment-4322126). 
Please [reach out](https://github.com/meilisearch/meilisearch/discussions/3280) if the [new limits](add link) impede your usage.
```